### PR TITLE
update icp ecosystem

### PR DESCRIPTION
**Pull request by letmejustputthishere**

### CODE REVIEW

**Comment by letmejustputthishere** [comment]
@jubos pls merge at your convience

File: data/ecosystems/i/ithreem.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title = "IThreeM - I3M"
+sub_ecosystems = []
+github_organizations = ["https://github.com/ithreem"]
+[[repo]]
+url = "https://github.com/ithreem/i3m-website"
```
Comment (jdesanto) [inline data/ecosystems/i/ithreem.toml:5]: This should be removed or made public. We cannot access it as is.

**Review by jdesanto — CHANGES_REQUESTED** [review]
Review requests are mostly to make repos public for indexing.

**Comment by letmejustputthishere** [comment]
> Review requests are mostly to make repos public for indexing.

i'm not sure i can follow this. can you elaborate pls?

File: data/ecosystems/d/dfinity.toml [hunk 2]
```diff
@@ -487,9 +494,15 @@ url = "https://github.com/dfinity/nodeprovider-scripts"
 [[repo]]
 url = "https://github.com/dfinity/oid-registry"
 
+[[repo]]
```
[the remark this answers is not in the document]
Comment (letmejustputthishere) [reply data/ecosystems/d/dfinity.toml:497]: this is public now

**Comment by letmejustputthishere** [comment]
@jdesanto i incorporated your requested changes :) lmk if you want anything else to change.

**Comment by jdesanto** [comment]
> > Review requests are mostly to make repos public for indexing.
> 
> i'm not sure i can follow this. can you elaborate pls?

Sorry this wasn't clear. Many repos were private and we cannot view them.

**Comment by letmejustputthishere** [comment]
ah, ok! but now it's good to merge, right? we made some changes to avoid private repos in the future

**Comment by letmejustputthishere** [comment]
hey @jdesanto can we please merge this PR, and afterwards [this](https://github.com/electric-capital/crypto-ecosystems/pull/1526)

**Comment by letmejustputthishere** [comment]
hey @yizhao-ec, can you merge this one first pls and then the one you just reviewed?

**Review by mariashen — APPROVED** [review]
Thank you for the PR!

**Pull request merged**